pac4j-core/pom.xml

```diff
@@ -40,6 +40,10 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I'm a bit reluctant to add this new library just for the `UriBuilder`...

Reply:
I see, sure. Do you want to add a utility method somewhere that can be used here? I really did not want to duplicate the parameter parsing logic. It's very difficult to get it right!

Reply:
Ah, absolutely sure but that only does the checking. Here we need the extraction. IOW I can check to see if a parameter is on the query string, but I'd have to parse the query string to extract it, and I find that prone to error. If you feel like it, I can still do it nonetheless.

Reply:
I think you can duplicate the logic I used in CAS: you can get the parameter with `webContext.getRequestParameter` and check if it is in the query string...